docs/node-operators/cli.md

```diff
@@ -94,74 +233,218 @@ witnet node blockchain --epoch=-10 --limit=1
+Deployed in block 159045c67d97e284588be4da89a43319735919144b2aaf1468e7eae6682f42b6 by twit10smy4dez7dpuc8yk3ck2qzgtuaa4vx3nqh5ysl
+FINISHED with 2 commits and 2 reveals
+Commit rounds: 1/2
+Reveal rounds: 1/2
+Reveals:
+    [Rewarded ] twit1xpc5d7dz6rsnnjrlp98vtn496qzrawu7y5cyx4: RadonTypes::RadonFloat(10172.642950000001)
+    [Rewarded ] twit1xcnfm3q57h7uh7y2m23y6hd7jsqufyrlmxgfnq: RadonTypes::RadonFloat(10172.642950000001)
+Tally: RadonTypes::RadonFloat(10172.642950000001)
 ```
-34099999950005
+
+NOTE: if you want to see the request stages in real time you can use the `watch` command.
+
+```sh tab="docker"
+watch --color docker exec -i witnet_node ./witnet node dataRequestReport 33c656101dd1dfa2258415d6487c648152b95983d1151f46a679b5d1902f6a8d
 ```
 
-### getPkh
+```sh tab="binary"
+watch --color witnet node dataRequestReport 33c656101dd1dfa2258415d6487c648152b95983d1151f46a679b5d1902f6a8d
+```
 
-Get the public key hash of the node.
+```sh tab="cargo"
+watch --color cargo run -- node dataRequestReport 33c656101dd1dfa2258415d6487c648152b95983d1151f46a679b5d1902f6a8d
+```
 
-This is the address used for mining blocks, resolving data requests, and
-receiving value transfer transactions.
+### joinTransaction
+
+Create a value transfer transaction that joins UTXOs.
+
+The following arguments must be provided.
 
-```sh
-witnet node getPkh
+* `--value=amount`: amount in nanoWits.
+* `--fee=fee`: fee.
+
+```sh tab="docker"
+docker exec -i witnet_node ./witnet node joinTransaction --value=100 --fee=1
 ```
 
-Example output:
+```sh tab="binary"
+witnet node joinTransaction --value=100 --fee=1
+```
 
+```sh tab="cargo"
+cargo run -- node joinTransaction --value=100 --fee=1
 ```
-twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr
+
+### knownPeers
+
+Get the list of peers known by the node.
+
+```sh tab="docker"
+docker exec -i witnet_node ./witnet node knownPeers
+```
+
+```sh tab="binary"
+witnet node knownPeers
 ```
 
-### getReputation
+```sh tab="cargo"
+cargo run -- node knownPeers
+```
 
-Get the reputation of the given account.
+### masterKeyExport
 
-* `--pkh=address`: Public key hash for which to get balance. If omitted, defaults to the node pkh.
+Export the node master key.
 
-```sh
-witnet node getReputation
-witnet node getReputation --pkh=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr
+```sh tab="docker"
+docker exec -i witnet_node ./witnet node masterKeyExport
 ```
 
-Example output:
+```sh tab="binary"
+witnet node masterkeyExport
+```
 
+```sh tab="cargo"
+cargo run -- node masterkeyExport
 ```
-Identity twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr has 1 reputation and is active
+
+### nodeStats
+
+Get the node stats.
+
+```sh tab="docker"
+docker exec -i witnet_node ./witnet node nodeStats
+```
+
+```sh tab="cargo"
+cargo run -- node nodeStats
 ```
 
-```sh
-witnet node getReputation --all
+```sh tab="binary"
+witnet node nodeStats
 ```
 
+The output shows information about your node :
```

Comment:
Among other information, this shows counters for proposed and accepted blocks and participations in resolving data requests ("commitments"):